ansible/files/adminapi.sudoers.conf

```diff
@@ -12,6 +12,7 @@ Cmnd_Alias PGBOUNCER = /bin/systemctl start pgbouncer.service, /bin/systemctl st
 %adminapi ALL= NOPASSWD: /root/manage_readonly_mode.sh
 %adminapi ALL= NOPASSWD: /usr/bin/systemctl daemon-reload
 %adminapi ALL= NOPASSWD: /usr/bin/systemctl restart postgresql.service
+%adminapi ALL= NOPASSWD: /usr/bin/systemctl restart pgbouncer
```

Comment:
already included - https://github.com/supabase/postgres/pull/386/files#diff-e032a7edf74375b338a9acdfa2cd14ce9d56ec2ad0d72e67c7eaa80234b2563aL4

Reply:
@darora, ok, will update the adminapi repo. Seems like `restart pgbouncer.service` and `restart pgbouncer` are treated differently.